
PackagesNotFoundError: The following packages are missing from the target environment:
  - cython


Building graph of deps:   0%|          | 0/6 [00:00<?, ?it/s]Examining setuptools:   0%|          | 0/6 [00:00<?, ?it/s]  Examining numpy=1.16:  17%|█▋        | 1/6 [00:02<00:10,  2.08s/it]Examining numpy=1.16:  33%|███▎      | 2/6 [00:02<00:04,  1.04s/it]Examining cython:  33%|███▎      | 2/6 [00:03<00:04,  1.04s/it]    Examining cython:  50%|█████     | 3/6 [00:03<00:03,  1.01s/it]Examining python=3.10:  50%|█████     | 3/6 [00:06<00:03,  1.01s/it]Examining python=3.10:  67%|██████▋   | 4/6 [00:06<00:03,  1.74s/it]Examining @/win-64::__win==0=0:  67%|██████▋   | 4/6 [00:06<00:03,  1.74s/it]Examining @/win-64::__win==0=0:  83%|████████▎ | 5/6 [00:06<00:01,  1.21s/it]Examining @/win-64::__archspec==1=x86_64:  83%|████████▎ | 5/6 [00:06<00:01,  1.21s/it]                                                                                       Determining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]Examining conflict for setuptools python cython:   0%|          | 0/6 [00:00<?, ?it/s]Examining conflict for setuptools numpy python cython:  17%|█▋        | 1/6 [00:00<00:00, 64.00it/s]Examining conflict for setuptools cython:  33%|███▎      | 2/6 [00:00<00:00,  7.11it/s]             Examining conflict for setuptools cython:  50%|█████     | 3/6 [00:00<00:00, 10.67it/s]Examining conflict for setuptools numpy cython:  50%|█████     | 3/6 [00:00<00:00, 10.67it/s]                                                                                             Building graph of deps:   0%|          | 0/6 [00:00<?, ?it/s]Examining setuptools:   0%|          | 0/6 [00:00<?, ?it/s]  Examining numpy=1.16:  17%|█▋        | 1/6 [00:02<00:10,  2.07s/it]Examining numpy=1.16:  33%|███▎      | 2/6 [00:02<00:04,  1.03s/it]Examining cython:  33%|███▎      | 2/6 [00:03<00:04,  1.03s/it]    Examining cython:  50%|█████     | 3/6 [00:03<00:03,  1.02s/it]Examining python=3.10:  50%|█████     | 3/6 [00:06<00:03,  1.02s/it]Examining python=3.10:  67%|██████▋   | 4/6 [00:06<00:03,  1.76s/it]Examining @/win-64::__win==0=0:  67%|██████▋   | 4/6 [00:06<00:03,  1.76s/it]Examining @/win-64::__win==0=0:  83%|████████▎ | 5/6 [00:06<00:01,  1.22s/it]Examining @/win-64::__archspec==1=x86_64:  83%|████████▎ | 5/6 [00:06<00:01,  1.22s/it]                                                                                       Determining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]Examining conflict for setuptools python cython:   0%|          | 0/6 [00:00<?, ?it/s]Examining conflict for setuptools numpy python cython:  17%|█▋        | 1/6 [00:00<00:00, 63.96it/s]Examining conflict for setuptools cython:  33%|███▎      | 2/6 [00:00<00:00,  7.50it/s]             Examining conflict for setuptools cython:  50%|█████     | 3/6 [00:00<00:00, 11.26it/s]Examining conflict for setuptools numpy cython:  50%|█████     | 3/6 [00:00<00:00, 11.26it/s]                                                                                             Returning non-final recipe for load_setup_py_test-0.1.0-np116py310_0; one or more dependencies was unsatisfiable:
Host: certifi, certifi, setuptools, setuptools, python, python, python, vs2008_runtime, vs2008_runtime, vs2008_runtime
deprecation warning: this function has been renamed to get_output_file_paths, to reflect that potentially multiple paths are returned.  This function will be removed in the conda-build 4.0 release.
